src/index.html

```diff
@@ -11,5 +11,9 @@
 	<body class="gutenberg-kit wp-embed-responsive">
 		<div id="root" class="gutenberg-kit-root"></div>
 		<script type="module" src="/index.jsx"></script>
+		<div
+			id="popover-fallback-container"
+			class="components-popover__fallback-container"
+		></div>
```

Comment:
Manually place a "fallback" popover element in the root so that we may query it via its `id` attribute.